apollo-router/src/plugins/rhai/engine.rs

```diff
@@ -4,6 +4,8 @@ use std::sync::Arc;
 use std::sync::Mutex;
 use std::time::SystemTime;
 
+use base64::prelude::BASE64_STANDARD;
+use base64::Engine as _;
```

Comment:
Same as my previous comment about the same side-effect module requirement as above.